contracts/PartnersRegistry.sol

```diff
@@ -0,0 +1,62 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.6.10;
+
+import "skill-wallet/contracts/main/ISkillWallet.sol";
+
+import "./PartnersAgreement.sol";
+import "./IDistributedTown.sol";
+import "./ICommunity.sol";
+
+contract PartnersRegistry {
+
+    event PartnersAgreementCreated(
+        address partnersAgreementAddress,
+        address communityAddress
+    );
+    IDistributedTown distributedTown;
+    address[] agreements;
+
+    constructor(address distributedTownAddress) public {
+        distributedTown = IDistributedTown(distributedTownAddress);
+    }
+
+    function create(
+        string memory metadata,
+        uint256 template,
+        uint256 rolesCount,
+        uint256 numberOfActions,
+        address partnersContractAddress,
+        uint256 membersAllowed
+    ) public {
+        require(
+            template >= 0 && template <= 2,
```

Comment:
Probably better to use enum from data types lib. E.g. in Q2T valid templates are 1-3 (0 is NONE, which is not valid but needed for some mappings)

Reply:
Noted. 
Let's another npm package with the common types, enums and constants and use them in all the product suites. 
In DistributedTown & SW they start from 0 as well, so we already have inconsistency. I'll write a task in the dito repo for that.